Source/Diagnostics/BoostedFrameDiagnostic.H

```diff
@@ -49,8 +48,7 @@ class LabFrameDiag {
     amrex::Real inv_gamma_boost_;
     amrex::Real inv_beta_boost_;
     amrex::Real dz_lab_;
-    amrex::Real dx_;
-    amrex::Real dy_;
+    amrex::Real particle_slice_dx_lab_;
```

Comment:
Could you prefix all member variables added in this PR with `m_`? We decided on this convention, but we forgot to add it to the CONTRIBUTING markdown. I made a PR for that, see https://github.com/ECP-WarpX/WarpX/pull/490.

Reply:
Added the prefix, m_ , to all the member variables.